### PR TITLE
EOS-22429: Added Alex workflow

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

# Problem Statement
Using Inclusive words are harmful

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: Y
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: Y
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: N
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N]: N
- [x] Jira is updated
- [ ] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [x] All the tests performed should be mentioned before Resolving a JIRA. 
- [x] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: .github/workflows/alex_reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline .github/workflows/alex_reviewdog.yml:2]: 2020 -> 2021

**Comment by sachinpunadikar** [comment]
Please fill in the PR template

File: .github/workflows/alex_reviewdog.yml (hunk 1 above)
Comment (venkuppu-chn) [reply .github/workflows/alex_reviewdog.yml]: Done

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**